workflows/ascc_organellar.nf

```diff
@@ -503,7 +486,7 @@ workflow ASCC_ORGANELLAR {
     softwareVersionsToYAML(ch_versions)
         .collectFile(
             storeDir: "${params.outdir}/pipeline_info",
-            name: 'nf_core_pipeline_software_mqc_versions.yml',
+            name: 'ascc_software_versions.yml',
```

Comment:
What happens with the interplay with this section and the same section in ASCC_GENOMIC? Do you end up with duplicate entries inside the versions file?

Reply:
Yeah i realised this when I changed the name of the file. There should really be an ascc.nf which then splits out to ascc_genomic and ascc_organellar. This will be a TODO.

Reply:
Testing now